### PR TITLE
Anitya dont rebuild already running jobs

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: frontend/coprs_frontend/run/check_for_anitya_version_updates.py [hunk 1]
```diff
@@ -80,12 +80,14 @@ def get_updated_packages(updates_messages, backend):
     for message in updates_messages:
         update = message['msg']
         project = update['project']
+        projectname = project['name'].lower()
```
Comment (FrostyX) [inline frontend/coprs_frontend/run/check_for_anitya_version_updates.py:67]: Not sure if we want to convert package names to lowercase but AFAIK PyPI packages are not case-sensitive so converting them or not converting them should both cause no problems.

I have no preference, I only noticed it

File: frontend/coprs_frontend/run/check_for_anitya_version_updates.py [hunk 2]
```diff
@@ -157,11 +157,15 @@ def is_prerelease(version: str) -> bool:
     """
     Detect a pre-release version string.
```
Comment (FrostyX) [inline frontend/coprs_frontend/run/check_for_anitya_version_updates.py:159]: I think the function name `is_prerelease` is fine but can you please update the docstring so it is clear that it filters both pre-release and post-release versions?

**Review by FrostyX — COMMENTED** [review]
Otherwise LGTM.
I can see there is a lot of tests and thank you for that.

File: frontend/coprs_frontend/run/check_for_anitya_version_updates.py (hunk 1 above)
Comment (praiskup) [reply frontend/coprs_frontend/run/check_for_anitya_version_updates.py:67]: Yea, mee neither - this was decided before - I now just moved the position of this code.

File: frontend/coprs_frontend/run/check_for_anitya_version_updates.py (hunk 2 above)
Comment (praiskup) [reply frontend/coprs_frontend/run/check_for_anitya_version_updates.py:159]: I was thinking about it, whether "post" isn't actually just another "pre" release :-)

Comment (praiskup) [reply frontend/coprs_frontend/run/check_for_anitya_version_updates.py]: Updated.

**Review by FrostyX — APPROVED** [review]
Thank you for the changes, LGTM

**Pull request merged**